routes.jsx

```diff
@@ -15,21 +16,38 @@ import Navbar from './components/navbar/navbar.jsx';
 import Footer from './components/footer/footer.jsx';
 
 const App = React.createClass({
+  getInitialState() {
+    return {
+      suppressSplashScreen: localstorage.getItem(`suppressSplashScreen`) === `true`
```

Comment:
To make it more obvious that this is a boolean, not a string, it might be worth putting this in parentheses.